packages/@sanity/default-layout/src/navbar/presence/PresenceMenu.tsx

```diff
@@ -0,0 +1,118 @@
+import {UserAvatar} from '@sanity/base/components'
+import {useGlobalPresence} from '@sanity/base/hooks'
+import {
+  Card,
+  AvatarStack,
+  MenuButton,
+  Menu,
+  MenuDivider,
+  MenuItem,
+  Button,
+  Text,
+  Box,
+  Stack,
+  MenuButtonProps,
+} from '@sanity/ui'
+import {CogIcon, UsersIcon} from '@sanity/icons'
+import React, {useMemo} from 'react'
+import styled from 'styled-components'
+import {versionedClient} from '../../versionedClient'
+import {StatusButton} from '../components'
+import {PresenceMenuItem} from '.'
+
+type PresenceMenuProps = {
+  collapse?: boolean
+  maxAvatars?: number
+}
+const AvatarStackCard = styled(Card)`
+  background: transparent;
+`
+
+const FooterCard = styled(Card)`
+  position: sticky;
+  bottom: 0;
+`
+
+const StyledMenu = styled(Menu)`
+  max-width: 320px;
+`
+
+const PRESENCE_MENU_POPOVER_PROPS: MenuButtonProps['popover'] = {
+  portal: true,
+  constrainSize: true,
+  scheme: 'light',
+}
+
+export function PresenceMenu({collapse, maxAvatars}: PresenceMenuProps) {
+  const {projectId} = versionedClient.config()
+  const presence = useGlobalPresence()
+  const hasPresence = presence.length > 0
+
+  const button = useMemo(
+    () =>
+      collapse ? (
+        <StatusButton
+          icon={UsersIcon}
+          mode="bleed"
+          statusTone={hasPresence ? 'positive' : undefined}
+        />
+      ) : (
+        <Button mode="bleed" padding={1}>
+          <AvatarStackCard>
+            <AvatarStack maxLength={maxAvatars}>
+              {presence.map((item) => (
+                <UserAvatar key={item.user.id} user={item.user} />
+              ))}
+            </AvatarStack>
+          </AvatarStackCard>
+        </Button>
+      ),
+    [collapse, maxAvatars, presence, hasPresence]
+  )
+
+  return (
+    <MenuButton
+      button={button}
+      popover={PRESENCE_MENU_POPOVER_PROPS}
+      id="presence-menu"
+      placement="bottom"
+      menu={
+        <StyledMenu padding={0}>
+          {!hasPresence && (
+            <Box paddingX={3} paddingY={4}>
+              <Stack space={3}>
+                <Text weight="semibold" size={2}>
+                  No one else is here
+                </Text>
+                <Text size={1} muted>
+                  Invite people to the project to see their online status.
+                </Text>
+              </Stack>
+            </Box>
+          )}
+          {hasPresence && (
+            <Box padding={1} paddingBottom={0}>
+              {presence.map((item) => (
+                <PresenceMenuItem presence={item} key={item.user.id} />
+              ))}
+            </Box>
+          )}
+          <FooterCard padding={1} paddingTop={0} radius={3}>
+            <Stack space={1}>
+              <MenuDivider />
+              <MenuItem
+                as="a"
+                text="Manage members"
+                paddingY={4}
```

Comment:
I think this would look better with no custom padding, actually :-)

Reply:
Not sure about that? :) 

<img width="299" alt="Screenshot 2021-09-22 at 10 00 30" src="https://user-images.githubusercontent.com/15094168/134305407-f54f155e-4ba6-4c2f-bc20-ff76eb27d88e.png">

Reply:
Hm, there's too much padding to the right of the link icons 👀

Reply:
```suggestion
                padding={4}
                paddingLeft={3}
```